### PR TITLE
Add more details about <block>

### PR DESCRIPTION
**Pull request by speedy008**
## Purpose of this pull request

This pull request (PR) contains more details about <block>

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/xml-instructions.html
- https://devdocs.magento.com/guides/v2.2/frontend-dev-guide/layouts/xml-instructions.html
- https://devdocs.magento.com/guides/v2.1/frontend-dev-guide/layouts/xml-instructions.html

whatsnew
Added a new paragraph that provides more context about the [purpose of blocks in layouts](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/xml-instructions.html).

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.1/frontend-dev-guide/layouts/xml-instructions.md [hunk 1]
```diff
@@ -40,6 +40,8 @@ Defines a block.
 
 **Details:** A block is a unit of page output that renders some distinctive content (anything visually tangible for the end-user), such as a piece of information or a user interface element.
 
+Blocks are a foundational building unit for layout in Magento. They are the linkbetween a PHP block class, which contains logic and a template which renders content. Blocks can have children and grandchildren (and so on). Information can be passed from layout XML into the block via the `<arguments/>` child node.
```
Comment (jeff-matthews) [inline guides/v2.1/frontend-dev-guide/layouts/xml-instructions.md]:
```suggestion
Blocks are a foundational building unit for layouts in Magento. They are the link between a PHP block class (which contains logic) and a template (which renders content). Blocks can have children and grandchildren (and so on). Information can be passed from layout XML files to blocks using the `<arguments/>` child node. 
```

File: guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md [hunk 2]
```diff
@@ -42,6 +42,8 @@ Defines a block.
 
 **Details:** A block is a unit of page output that renders some distinctive content (anything visually tangible for the end-user), such as a piece of information or a user interface element.
 
+Blocks are a foundational building unit for layout in Magento. They are the linkbetween a PHP block class, which contains logic and a template which renders content. Blocks can have children and grandchildren (and so on). Information can be passed from layout XML into the block via the `<arguments/>` child node.
```
Comment (jeff-matthews) [inline guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md]:
```suggestion
Blocks are a foundational building unit for layouts in Magento. They are the link between a PHP block class (which contains logic) and a template (which renders content). Blocks can have children and grandchildren (and so on). Information can be passed from layout XML files to blocks using the `<arguments/>` child node. 
```

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by jeff-matthews** [comment]
running tests

**Comment by jeff-matthews** [comment]
running tests

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**